test/integration/Turing/runtests.jl

```diff
@@ -1,7 +1,15 @@
-using Pathfinder, Random, Test, Turing
+using LogDensityProblems, LinearAlgebra, Pathfinder, Random, Test, Turing
+using Turing.Bijectors
+
+if isdefined(Base, :get_extension)
+    PathfinderTuringExt = Base.get_extension(Pathfinder, :PathfinderTuringExt)
+else
+    PathfinderTuringExt = Pathfinder.PathfinderTuringExt
+end
 
 Random.seed!(0)
 
+#! format: off
```

Comment:
Would be curious to understand why this is needed.

Reply:
Otherwise JuliaFormatter prepends the final sampling statement in each model with a `return`, which looks funny.